tests/include/test/drivers/test_driver_common.h

```diff
@@ -66,11 +66,7 @@ typedef psa_crypto_driver_pake_step_t libtestdriver1_psa_crypto_driver_pake_step
  * - In TF-PSA-Crypto, the libtestdriver1 library code is located in
  *   `drivers/libtestdriver1`. The header path is relative to
  *   `drivers/libtestdriver1/include` and has the form:
- *     `../../libtestdriver1/src/xyz`
- *
- *   Note: the path cannot simply be `../src/xyz`, because that could refer to
- *   a header under `drivers/builtin/src`. The directory `drivers/builtin/include`
- *   is also in the header search path when compiling test drivers.
```

Comment:
I lost track of why “the path cannot simply be `../src/xyz`” no longer applies. It's because the headers are now renamed with a prefix, e.g. `<mbedtls/private/libtestdriver1-aes.h>`, whereas in the original design they were just copied to e.g. `<mbedtls/private/aes.h>` so you had to make sure not to land on the one under `builtin`. This is only relevant to the history within this PR, so it doesn't warrant a comment in the file, but I'm leaving it in writing in case we have doubts later.